workspaces/ui/src/services/TestingService.js

```diff
@@ -144,6 +166,24 @@ export async function createExampleTestingService(exampleId = 'todo-report') {
   return new ExampleTestingService(orgId);
 }
 
+export class TestingServiceError extends Error {
+  static type = 'testing-service-error';
+  // don't rely on `instanceof` in JS, it's a mistake
```

Comment:
Interesting, I'll have to double check what our typescript code is generating too. https://github.com/Microsoft/TypeScript-wiki/blob/master/Breaking-Changes.md#extending-built-ins-like-error-array-and-map-may-no-longer-work